### DISCUSSION TITLE
Scrolling with arrows press (key up, key down) does not work after loading the page without clicking the mouse in window

### DISCUSSION THREAD

**Discussion by MattCode-dev** [idiotWu/smooth-scrollbar#484] [Q&A] [answered]
Hi, I have a question about how smooth-scroll works.

At this point, it is difficult to use the website without a mouse and a touchpad in a situation where we want to move the page down by pressing the key down - nothing happens.

You have to click on the site with the mouse and only then you can use the key up, down or space.

Question:

What to do to make the key up, key down, space buttons work immediately after loading the page?

**Comment by dolphin-wood** [comment, marked as answer]
Keyboard events require the scrollbar container to be focused, so after loading the page, call

```js
container.focus()
```

**Reply by MattCode-dev** [reply]
It works, thank you

**Comment by Nikolasgrizli** [comment]
Hi everyone, I tried to implement this tip in my react app, but it didn't work out((( 

```
useEffect(() => {
    const wrapper = scrollbar.current;
    if (wrapper) {
      wrapper.focus();
    }
  }, []);
```
